### PR TITLE
Consolidate all the node down/reboot/disconnect and kubelet restart manual test …

### PR DESCRIPTION
**Pull request by roger-ryao**
…cases

This commit adds the consolidated manual test cases for node `down`/`reboot`/`disconnect` and `kubelet restart` scenarios. The related cases were found on https://longhorn.github.io/longhorn-tests/manual/pre-release/ and made generic to be moved to one place. This will improve test case management and make it easier to maintain in the future.

Ref: https://github.com/longhorn/longhorn/issues/5569

Signed-off-by: Roger Yao <roger.yao@suse.com>

### CODE REVIEW

**Review by khushboo-rancher — COMMENTED** [review]
In general, LGTM. we can add node disconnection cases as well.

**Comment by roger-ryao** [comment]
> In general, LGTM. we can add node disconnection cases as well.

Add node disconnection cases and update the [PR](https://github.com/longhorn/longhorn-tests/compare/53e3c79f87986c6a4f822c0bd905bef4c1dc8219..89b0f4e98f5694b0713024f0aa6919a5374b8717)

**Review by khushboo-rancher — APPROVED** [review]
Thanks @roger-ryao

File: docs/content/manual/pre-release/node-not-ready/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: Node Not Ready
+---
+Consolidate test cases for [#5569 [TEST] node down/reboot and kubelet restart](
+https://github.com/longhorn/longhorn/issues/5569)
```
Comment (khushboo-rancher) [inline docs/content/manual/pre-release/node-not-ready/_index.md:5]: I missed this before merging.
@roger-ryao can you please remove these lines, it's not needed?

Comment (roger-ryao) [reply docs/content/manual/pre-release/node-not-ready/_index.md:5]: > I missed this before merging. @roger-ryao can you please remove these lines, it's not needed?

Hi @khushboo-rancher 
      [PR-1343](https://github.com/longhorn/longhorn-tests/pull/1343) is ready for review.

**Pull request merged**